src/art/jippy89/script.js

```diff
@@ -0,0 +1,93 @@
+let viewWidth, viewHeight
+
+function resizeCanvas (canvas) {
+  // console.log('resizeCanvas')
```

Comment:
Please remove outdated code.

Reply:
Hi @sophiabrandt, do you mean the console logs? If yes, I just removed it. Thanks!

Reply:
I have removed `viewWidth` and `viewHeight` since I could use the `width` and `height` from the `canvas` object instead.